JSDemos/Demos/Gauges/GaugeTitleCustomized/React/description.md

```diff
@@ -0,0 +1,11 @@
+This demo shows how to specify the CircularGauge title and display custom labels in the component's center. 
+
+The CircularGauge title usually contain general explanation about the data represented by the gauge. You can configure it with the properties of the [title](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/) configuration object:
+
+- **Location**    
+Use the [horizontalAlignment](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/#horizontalAlignment) and [verticalAlignment](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/#verticalAlignment) properties to place a title on any side of your gauge.
+
+- **Font Settings**    
+To customize a text appearance, define the font properties within the [font](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/font/) configuration object.
```

Comment:
"to customize text appearance, define font properties" - you basically trying to repeat the words from the title (Font Settings). I already know that this section will describe how to change font settings. So I can simply refer to the configuration object and that would be enough. 

```suggestion
Use the [font](/Documentation/ApiReference/UI_Components/dxCircularGauge/Configuration/title/font/) configuration object.
```